Makefile

```diff
@@ -298,6 +298,7 @@ publish: packages
 lint: ## run linters
 	go version
 	golangci-lint version
+	GO111MODULE=on golangci-lint cache clean -v
```

Comment:
won't this make linting incredibly slow if we always clean the cache?

Reply:
one argument is that linting should be slow on CI in order to be complete. I don't have a strong opinion though. I added this when I was struggling with some differences I was seeing on CI vs local but it ended up being something different. I'm happy to remove.